apps/server/src/controllers/sessionContorller.js

```diff
@@ -44,4 +32,12 @@ module.exports = {
       res.status(500).json(err);
     }
   },
+  async getAllSessions( req, res) {
+    try {
+      let session = await Session.find({userID: req.query.id}, {}, { sort: {'startTime' : -1}});
```

Comment:
@AMess33 Looks like this part is working correctly, the only problem is sessions don't currently have a `userID` so it's not finding any and returning an empty array. Need to get that on the model and pass it in with the request when creating.